integration.Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM hubbleprotocol/anchor-build:0.25.0
```

Comment:
why don't we fetch most of the `.so`s from mainnet, only keep the necessary ones?

Reply:
we do, but we need the `solana` runtime and `solana-local-validator`, that are already set up in the docker image